server/utils/timeline.ts

```diff
@@ -43,7 +43,7 @@ export const renderTimelineEventContent = (event: Cas1TimelineEvent): string =>
         const isoDateToUiDateOrUndefined = (isoDate: string) =>
           isoDate ? DateFormats.isoDateToUIDate(isoDate) : undefined
         const roomCriteriaOrNone = (criteria: Array<Cas1SpaceCharacteristic>) =>
-          characteristicsInlineList(filterRoomLevelCriteria(criteria || [])) || 'none'
+          characteristicsInlineList(filterRoomLevelCriteria(criteria || []), 'none')
```

Comment:
I was wondering about suggesting moving the `filterRoomLevelCriteria` filter within `characteristicsInlineList` but I guess it's needed for changing types so it's nicer to be explicit like this.